### PR TITLE
Parse both json and yml

### LINKED ISSUE #26: Accept Dogfiles in JSON
YAML is a superset of JSON, so any Dogfile written in JSON should be valid, we should support even Dogfiles mixing JSON and YAML

### PR DESCRIPTION
**Pull request by n-marshall**
This closes #26

### CODE REVIEW

File: parser/parser.go [hunk 1]
```diff
@@ -106,18 +103,28 @@ func LoadDogFile() (tm types.TaskMap, err error) {
...
 
-	return ParseDogfile(d)
+	// for _, dogfile := range dogfiles {
+	// 	var fileData []byte
+	// 	fileData, err = ioutil.ReadFile(dogfile.Name())
+	// 	if err != nil {
+	// 		return
+	// 	}
+	// 	d = append(d, fileData...)
+	// }
```
Comment (xsb) [inline parser/parser.go]: This block is commented.

Is that on purpose for future usage? I haven't tried it.

Comment (n-marshall) [inline parser/parser.go]: Nope, this should definitely be going away, it's how we used to concatenate files. Let me fix it quickly

Comment (n-marshall) [inline parser/parser.go]: done

**Comment by xsb** [comment]
Thanks @n-marshall !

**Pull request merged**